internal/store/resourcesync.go

```diff
@@ -97,10 +101,40 @@ func (s *ResourceSyncStore) List(ctx context.Context, orgId uuid.UUID, listParam
 	return &apiResourceSyncList, result.Error
 }
 
-func (s *ResourceSyncStore) DeleteAll(ctx context.Context, orgId uuid.UUID) error {
-	condition := model.ResourceSync{}
-	result := s.db.Unscoped().Where("org_id = ?", orgId).Delete(&condition)
-	return result.Error
+func (s *ResourceSyncStore) DeleteAll(ctx context.Context, orgId uuid.UUID, callback removeOwnerCallback) error {
```

Comment:
Can we simplify this by removing the owner for all fleets in the org where the owner is "ResourceSync/?"

Reply:
We can, The only thing is that this will bypass the fleet store. Bypassing callbacks/validations  (whatever) when we add them, etc.